sys/linux/fs_ioctl_ext4.txt

```diff
@@ -48,3 +50,9 @@ move_extent {
 	len		int64
 	moved_len	int64
 }
+
+fsuuid {
+	fsu_len		len[fsu_uuid, int32]
+	fsu_flags	int32
```

Comment:
`const[0, int32]`
The code fails if it's not 0:
https://elixir.bootlin.com/linux/latest/source/fs/ext4/ioctl.c#L1197

Reply:
Yes, it's always 0 right now. I thought it could change later and KCOV_ENABLE_COMPARISONS should help to pass current `== 0` relatively fast.

Reply:
0 should be passed relatively easily as one of special values. However, the fuzzer will still spend randomness and mutation iterations on all other values.
We generally describe the current state of the interface.